src/java.base/share/classes/jdk/internal/crac/JDKResource.java

```diff
@@ -29,72 +29,4 @@
 import jdk.crac.Resource;
 
 public interface JDKResource extends Resource {
```

Comment:
Could we drop the interface completely?

Reply:
Probably, I propose to do this a bit later. E.g. it may be useful to override beforeCheckpoint here declaring no Exception is thrown.